packages/components/src/Popover/types.ts

```diff
@@ -0,0 +1,21 @@
+export type Variant =
+  | "default"
+  | "informative"
+  | "positive"
+  | "negative"
+  | "cautionary"
+
+export type Size = "small" | "large"
+
+export const PlacementKeys = [
```

Comment:
lowerCamelCase and plural since it's a variable and not a Component nor Type.

```suggestion
export const popoverPlacements = [
```

Although technically I think the convention for constants is actually UPPER_SNAKE, so `POPOVER_PLACEMENTS`? 🤔 
https://javascript.info/variables#uppercase-constants